source/_components/switch.mystrom.markdown

```diff
@@ -11,6 +11,7 @@ logo: mystrom.png
 ha_category: Switch
 ha_release: 0.9
 ha_iot_class: "Local Polling"
+ha_qa_scale: silver
```

Comment:
Ah, this is an interesting one. I think that we should make silver level include that authentication is done at a component level, not a platform level. That will prevent breaking changes in the future.

Reply:
This as well. I will remove both when the link is updated. Sorry, for the confusion.